index.js

```diff
@@ -5,11 +5,11 @@ export function hiMom(motherName, motherLang = "en") {
     throw new Error("Language not yet supported, but hi mom anyway!");
   }
 
-  let re = /{{([\s\S]+)}}/;
-  let greeting = intlMom[motherLang];
-  let mom = greeting.match(re)[0].slice(2, -2);
+  const regex = /\{{2}([^}]+)\}{2}/;
+  const greeting = intlMom[motherLang];
+  const momName = greeting.split(regex)[1];
```

Comment:
I'd consider different variable names here - motherName vs momName is confusing. If I understand it correctly, motherName is a user provided name eg "Alice" and mom is the localized word for "mom" eg "maman" (fr)

Maybe givenName and localizedWordForMom. Or givenMom and genericMom.

Reply:
we can rename parameters to `name` and `locale`, and add the JSDoc comment for describing them.

because parameters are shown in signature when user calls the function, so it makes more sense for function called `hiMom` to have parameters `name` and `locale`.

Reply:
or, instead of `locale`, it can be `language`

Reply:
Great, guys I will act on these improvements!

Reply:
About the mother's name that comes from inside the regex expression, I believe that `localeName` might work.

Reply:
if continuing @bobtista thoughts, maybe localizedName will be even better